lib/node_modules/@stdlib/array/base/mskunary5d/test/test.js

```diff
@@ -0,0 +1,404 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var zeros5d = require( '@stdlib/array/base/zeros5d' );
+var mskunary5d = require( '@stdlib/array/base/mskunary5d/lib' );
```

Comment:
```suggestion
var mskunary5d = require( './../lib' );
```
This path must be relative.